test/e2e/basic/available.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configv1 "github.com/openshift/api/config/v1"
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+)
+
+var _ = Describe("Node Tuning Operator: availability", func() {
+	It(fmt.Sprintf("clusteroperator/%s available", tunedv1.TunedClusterOperatorResourceName), func() {
```

Comment:
I'm not sure what is more beneficial, using the constant or just a plain string title. 
One pro for the plain string is it would be easier to find when tests fail.

Reply:
I see what you're saying and it makes sense.  However, if the "default" CR is ever changed you wouldn't need to go through the entire operator code to make appropriate changes.  Just change it in one place.  Also, when the tests fails, it gives you the line and the source file so I don't see locating where the test fails as an issue.  I'd rather leave things as they are.